app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+	2008-2010 Jonathan Beck
+	2009 Nikias Bassen
+	2008 Matt Colyer"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/libplist-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86 !x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	libplist$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:plistutil$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist$secondaryArchSuffix = 3.0.0 compat >= 3
+	lib:libplist++$secondaryArchSuffix = 3.0.0 compat >= 3
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libplist${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libplist$secondaryArchSuffix = 3.0.0 compat >= 3
+	devel:libplist++$secondaryArchSuffix = 3.0.0 compat >= 3
+	"
+REQUIRES_devel="
+	libplist$secondaryArchSuffix == $portVersion base
+	devel:libxml2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:python
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sh
+	"
+
+BUILD()
+{
+	runConfigure ./configure
```

Comment:
Can you change **`BUILD()`** to make it look like this:
```
BUILD()
{
	if [ -n "$secondaryArchSuffix" ]; then
		maybe_without_cython=--without-cython
	else
		maybe_without_cython=
	fi

	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir \
		$maybe_without_cython

	make $jobArgs
}
```

IMPORTANT: I don't know if passing **`--without-cython `** to configure is a good idea.
I had to do that because otherwise it would not build on x86 secondary arch.

But maybe we should depend on **`cmd:python$secondaryArchSuffix`** and avoid the **`--without-cython `**.
Let's wait until someone else can help us with this issue.